src/routes/home/HomeView.vue

```diff
@@ -198,6 +198,11 @@ export default {
     await this.gameListStore.requestGameList();
     this.loadingData = false;    
   },
+  mounted() {
+    if (history?.state?.error) {
```

Comment:
@seriousySean I used the history state to pass the error message on redirect in `router.js`

`return {name: 'Home', state: {error: err.message} };`

Reply:
I don't see the `state` in their documentation, only `query` or `params`. I'm certainly not against it if it's a standard usage of the router api, just want to make sure as I haven't seen this particular use case before. It seems like a query param might serve the same purpose without needing to use state at all, then you'd be able to remove the query param on mounted of the component if it exists.

Reply:
Yea, I'm not 100% sure to be honest which is the best way, I do feel like adding it as a query or param is a little weird

Reply:
Yeah, I get what you’re saying but query param is at least a standard usage and pretty normal in my experience. If you can find me documentation on that being a standard or expected use of Vue router I’m totally down, otherwise I’d like you to switch it.

Reply:
What about a conditional route param? Would that work in this case instead? Trying to find a decent alternative while sticking with some semblance of standards is tricky :)

Reply:
Last note, I promise! It's also ok to ship it and resolution later if you just want it out there. I would make sure to test in all browsers-- at least Chrome, Firefox and Safari (desktop + mobile) as they may behave differently with the api as called out in the RFC.

Reply:
@Haviles04 is there a reason that a query param would be more difficult to use?

Reply:
@seriouslysean after talking to @Haviles04 about it, one thing that I think is nice about using history state over a route param is that refreshing the page on the home view after getting the error will not re-trigger the error, without needing extra cleanup logic to clear the route param.

Given that the approaches are otherwise comparable, I think that clinches it for state

Reply:
So I ended up pushing it to be a query param, I was having an issue on displaying the error message if you were already on '/' and the router was redirecting you back to '/' because the component wouldn't reload which wouldn't fire the mounted hook. The only way around this was to force a refresh of the page before redirecting it which felt clunky to me. 

I tried to `watch` the history but it didn't fire, so instead I watched the `$route`, and show the snackbar anytime there is an error property in the `router.query`